### PR TITLE
Edit

### PR DESCRIPTION
**Pull request by munjalSaksham71**
In this PR: 

I have added edit functionality to a note with rich-text-editor

YOUR CAN CREATE A NEW ACCOUNT BY CLICKING ON ```REgister here``` and it will redirect to at signup form OR

Use these credential:  
Email: test@gmail.com
Password: 12345678

### CODE REVIEW

File: src/components/Modal/Modal.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+.modal_title {
+    width: 30rem;
+    height: 1rem;
+    padding: 10px;
+    font-weight: 500;
+    font-size: large;
+    border: 1px solid #000
+}
+
+.modal_note {
```
Comment (t007rushi) [inline src/components/Modal/Modal.css:42]: liked consistent naming

File: src/components/ViewNote/ViewNote.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import "./ViewNote.css";
-import { AiFillDelete } from "react-icons/ai";
+import { AiFillDelete, AiFillEdit } from "react-icons/ai";
 import { VscPinned } from "react-icons/vsc";
 import { IoIosColorPalette } from "react-icons/io";
 import { BiArchiveIn } from "react-icons/bi";
```
Comment (t007rushi) [inline src/components/ViewNote/ViewNote.jsx:5]: can make icons.js and import from there

**Review by t007rushi — APPROVED** [review]
unable to authorize, How to login ? also after signup find hard
ADD GUEST LOGIN

**Comment by munjalSaksham71** [comment]
> unable to authorize, How to login ? also after signup find hard ADD GUEST LOGIN

Hey Rushikesh, I can easily signup - there's a ```Register Here``` and click on that . and enter fields you will be redirected to homepage

File: src/components/ViewNote/ViewNote.jsx (hunk 2 above)
Comment (munjalSaksham71) [reply src/components/ViewNote/ViewNote.jsx:5]: will do it in next PR.

**Review by mariyasada — APPROVED** [review]
liked that you have created a separate context for the modal and the modal for edit functionality is also looks good

**Pull request merged**